### PR TITLE
Added more phrases

### PR DESCRIPTION
**Pull request by Vyvy-vi**
# Pull Request
More Phrases

## Description:

Adds some more staff, phrases for vonage and digitalocean, and a signing off message... ._. 
*about the signing off message... is it a good idea? (also this would work right? )
 

## Related Issue:
wupps forgot to open one... Sowwy

Closes #XXXXX
## Context: 
Screenshot 2020-10-25 at 10 15 17 AM
Screenshot 2020-10-25 at 10 15 48 AM

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -47,11 +62,63 @@ const isNick = (message: Message): boolean => {
...
   );
 };
 
+const isBen = (message: Message): boolean => {
+  return (
+    message.content.replace(/\s/g, "").toLowerCase().includes("ben") ||
+    message.content.replace(/\s/g, "").toLowerCase().includes("benjamin") ||
+    message.author.id === "664240791861985320" ||
+    message.content.includes("B E N J A M I N") ||
+    message.content.includes("BEN")
```
Comment (nhcarrigan) [inline src/index.ts:78]:
```suggestion
    message.content.includes("B E N")
```
If these are the unicode versions (for the regional indicator emoji) they need to be space separated.

File: src/index.ts [hunk 2]
```diff
@@ -47,11 +62,63 @@ const isNick = (message: Message): boolean => {
...
+    message.author.id === "135865279149375489" ||
+    message.content.includes("K Y L E")
+  );
+};
+
+const isJoe = (message: Message): boolean => {
+  return (
+    message.content.replace(/\s/g, "").toLowerCase().includes("joe") ||
+    message.author.id === "163355465793798145" ||
+    message.content.includes("JOE")
```
Comment (nhcarrigan) [inline src/index.ts:110]:
```suggestion
    message.content.includes("J O E")
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

**Review by nhcarrigan — APPROVED** [review]
LGTM~! 🥳

**Comment by Vyvy-vi** [comment]
um hey can you do a small edit
Screenshot 2020-10-25 at 12 51 36 PM
```
ToSay = "True"
```

**Pull request merged**